src/TestcontainersCloud.test.js

```diff
@@ -1,37 +1,83 @@
-const redis = require('async-redis');
-const { GenericContainer } = require('testcontainers');
+const { PostgreSqlContainer} = require('@testcontainers/postgresql');
 const Docker = require('dockerode');
+const {fail} = require("assert");
+
+const logo = "\n" +
+    "████████╗███████╗███████╗████████╗ ██████╗ ██████╗ ███╗   ██╗████████╗ █████╗ ██╗███╗   ██╗███████╗██████╗ ███████╗ \n" +
+    "╚══██╔══╝██╔════╝██╔════╝╚══██╔══╝██╔════╝██╔═══██╗████╗  ██║╚══██╔══╝██╔══██╗██║████╗  ██║██╔════╝██╔══██╗██╔════╝ \n" +
+    "   ██║   █████╗  ███████╗   ██║   ██║     ██║   ██║██╔██╗ ██║   ██║   ███████║██║██╔██╗ ██║█████╗  ██████╔╝███████╗ \n" +
+    "   ██║   ██╔══╝  ╚════██║   ██║   ██║     ██║   ██║██║╚██╗██║   ██║   ██╔══██║██║██║╚██╗██║██╔══╝  ██╔══██╗╚════██║ \n" +
+    "   ██║   ███████╗███████║   ██║   ╚██████╗╚██████╔╝██║ ╚████║   ██║   ██║  ██║██║██║ ╚████║███████╗██║  ██║███████║ \n" +
+    "   ╚═╝   ╚══════╝╚══════╝   ╚═╝    ╚═════╝ ╚═════╝ ╚═╝  ╚═══╝   ╚═╝   ╚═╝  ╚═╝╚═╝╚═╝  ╚═══╝╚══════╝╚═╝  ╚═╝╚══════╝ \n" +
+    "  \n" +
+    "  Congratulations on running your first test! 🎉\n" +
+    "  Runtime used: \n" +
+    "      ::::::\n" +
+    " \n" +
+    "  You can now return to the website to complete your onboarding.\n" +
+    " \n" +
+    "";
+
+const ohNo = "" +
+    " ██████╗ ██╗  ██╗    ███╗   ██╗ ██████╗               ██╗\n" +
+    "██╔═══██╗██║  ██║    ████╗  ██║██╔═══██╗    ██╗      ██╔╝\n" +
+    "██║   ██║███████║    ██╔██╗ ██║██║   ██║    ╚═╝█████╗██║ \n" +
+    "██║   ██║██╔══██║    ██║╚██╗██║██║   ██║    ██╗╚════╝██║ \n" +
+    "╚██████╔╝██║  ██║    ██║ ╚████║╚██████╔╝    ╚═╝      ╚██╗\n" +
+    " ╚═════╝ ╚═╝  ╚═╝    ╚═╝  ╚═══╝ ╚═════╝               ╚═╝\n" +
+    "                                                         ";
 
 describe('GenericContainer', () => {
-    let container;
-    let redisClient;
+    it('tcc cloud engine', async () => {
+        const docker = new Docker();
+        const info = await docker.info();
+        const serverVersion = info.ServerVersion;
+        const isTestcontainersDesktop = serverVersion.includes('Testcontainers Desktop');
+        const isTestcontainersCloud = serverVersion.includes('testcontainerscloud');
+        if (!(isTestcontainersDesktop || isTestcontainersCloud)) {
+            console.log(ohNo)
+            fail()
+        }
 
-    beforeAll(async () => {
-        container = await new GenericContainer('redis')
-            .withExposedPorts(6379)
-            .start();
+        let runtimeName = "Testcontainers Cloud";
+        if (!serverVersion.includes("testcontainerscloud")) {
+            runtimeName = serverVersion.os;
+        }
+        if (serverVersion.includes("Testcontainers Desktop")) {
+            runtimeName = "via Testcontainers Desktop app";
```

Comment:
please add space before "via" -> " via" it's a concatenated string, and it looks better with a space.